io/net/netperf_test.py

```diff
@@ -98,6 +98,7 @@ def setUp(self):
                                             password=self.peer_password)
         self.peer_public_networkinterface = NetworkInterface(self.peer_interface,
                                                              self.remotehost_public)
+        self.networkinterface.bring_up()
```

Comment:
@cforno12  everything  is fine. but why its needed here ..?

Reply:
self.networkinterface.bring_up() is already called in setup at line 66